linkml_runtime/utils/schemaview.py

```diff
@@ -48,31 +48,33 @@
     from collections.abc import Mapping
     from types import NotImplementedType
 
-    from linkml_runtime.utils.metamodelcore import URIorCURIE
+    from linkml_runtime.utils.metamodelcore import URI, URIorCURIE
 
 
 logger = logging.getLogger(__name__)
 
 MAPPING_TYPE = str  ## e.g. broad, exact, related, ...
 CACHE_SIZE = 1024
 
-SLOTS = "slots"
 CLASSES = "classes"
 ENUMS = "enums"
-SUBSETS = "subsets"
 TYPES = "types"
+SUBSETS = "subsets"
+ELEMENTS = [CLASSES, SLOTS, ENUMS, SUBSETS, TYPES]
+PREFIXES = "prefixes"
```

Comment:
Why are you declaring a new constant that is not being used?

Reply:
It gets used in a future commit. 🙂